app/views/courses/CoursesView.coffee

```diff
@@ -242,6 +243,15 @@ module.exports = class CoursesView extends RootView
     levelsUrl = @urls.courseWorldMap({course, courseInstance})
     application.router.navigate(levelsUrl, { trigger: true })
 
+  onClickViewRanking: (e) ->
+    courseID = $(e.target).data('course-id')
+    courseInstanceID = $(e.target).data('courseinstance-id')
+    window.tracker?.trackEvent 'Students View Ranking', category: 'Students', courseID: courseID, courseInstanceID: courseInstanceID, ['Mixpanel']
```

Comment:
Do we need to be tracking this event?

Reply:
Hi, Andrew. Yes, this function is required to call the event of the ranking link.
![WhatsApp Image 2019-05-16 at 17 02 27](https://user-images.githubusercontent.com/7550572/57883713-6bc4c680-77fd-11e9-9438-b9f38e81c360.jpeg)

Reply:
Is this tracking for you to collect information? I think I'm a little confused for its function.
Thanks for the screenshot!

Reply:
Yes, exactly. I'm testing the ranking in only one class. He still needs improvement. When it completes it can make available to the entire codecombat. Below is the ranking screen for test class.
If necessary I can give you the access data of a test account that can see this ranking.
![Screenshot_2019-05-16 CodeCombat Learn to Code by Playing a Game(4)](https://user-images.githubusercontent.com/7550572/57884687-9fa0eb80-77ff-11e9-9097-1d35dc6f41eb.png)

Reply:
Do you want to track the clicks on the button `.view-ranking-link`?
I believe that `window.tracker?.trackEvent` is to track an event internally for internal analysis, eg: to track an event on google analytics, which might not be required for your use case.

Reply:
Yes, that's it.
You suggest that I remove only the line 249 `window.tracker?.trackEvent 'Students View Ranking', category: 'Students', courseID: courseID, courseInstanceID: courseInstanceID, ['Mixpanel']`,  correct?